plugins/playlist/src/components/PlaylistIndexPage/PlaylistIndexPage.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  PageWithHeader,
+  Content,
+  ContentHeader,
+  SupportButton,
+} from '@backstage/core-components';
+import { CatalogFilterLayout } from '@backstage/plugin-catalog-react';
```

Comment:
I see that there's likely space for this filter layout system to be decoupled from the catalog and extracted into its own package :grin:

Reply:
Totally agree ! along with the list provider hook stuff too which I mostly plagiarized 😅 
All that can definitely be moved out and made more generic to be reused in multiple plugins (but probably in a future change since this PR is large enough already 😁)